content/de/faq.md

```diff
@@ -23,8 +23,8 @@ Zusätzlich gibt es mit [Nodebug.me](http://nodebug.me/) eine Möglichkeit dabei
 
 ## Wo finden Diskussionen statt?
 
-Es gibt einen #io.js-Kanal auf Freenode IRC. Diskussionslogs dieses Kanals gibt es [hier](http://logs.libuv.org/io.js/latest).
+Es gibt einen #io.js-Kanal auf dem Freenode IRC. Diskussionslogs dieses Kanals gibt es [hier](http://logs.libuv.org/io.js/latest).
 
 ## Was ist Open-Source Governance?
 
-Open source governance advocates the application of the philosophies of the open source and open content movements in order to enable any interested party to add to the creation of the end product, as with a wiki document. Legislation is democratically opened to the general citizenry, employing their collective wisdom to benefit the decision-making process and improve democracy. [[source]](https://en.wikipedia.org/wiki/Open-source_governance)
+Das Open-Source Aufsichtsmodell befürwortet die Teilnahme an den Open-Source- und Open-Content-Bewegungen, damit jeder Interessierte an dem Endprodukt teilhaben kann, so wie bei einem Wiki-Eintrag. Die Gesetzgebung ist demokratisch jedem geöffnet, um sich so den generellen Consensus zu Nutzen zu machen, wovon die Enscheidungskraft und die Demokratie des Projektes profitieren. [[Quelle in Englisch]](https://en.wikipedia.org/wiki/Open-source_governance)
```

Comment:
the problem is that I'm currently on vacation till Saturday, but I'll update it right then.

On 12 Jul 2015 06:03 +0200, Frederic Hemberger<notifications@github.com>, wrote:
>  
> Incontent/de/faq.md(https://github.com/nodejs/iojs.org/pull/394#discussion_r34418481):
>  
> >>## Was ist Open-Source Governance?>>-Open source governance advocates the application of the philosophies of the open source and open content movements in order to enable any interested party to add to the creation of the end product, as with a wiki document. Legislation is democratically opened to the general citizenry, employing their collective wisdom to benefit the decision-making process and improve democracy. [[source]](https://en.wikipedia.org/wiki/Open-source_governance)>+Das Open-Source Aufsichtsmodell befürwortet die Teilnahme an den Open-Source- und Open-Content-Bewegungen, damit jeder Interessierte an dem Endprodukt teilhaben kann, so wie bei einem Wiki-Eintrag. Die Gesetzgebung ist demokratisch jedem geöffnet, um sich so den generellen Consensus zu Nutzen zu machen, wovon die Enscheidungskraft und die Demokratie des Projektes profitieren. [[Quelle in Englisch]](https://en.wikipedia.org/wiki/Open-source_governance)
>  
> Can you please update your PR so we can get this merged? Thanks.
>  
>  
> —
> Reply to this email directly orview it on GitHub(https://github.com/nodejs/iojs.org/pull/394/files#r34418481).
>  
>  
>